bundles/framework/admin-announcements/view/AnnouncementsHelper.js

```diff
@@ -0,0 +1,87 @@
+export const announcementsHelper = () => {
```

Comment:
Consider making this an object instead of function since the function only returns an object.